src/Init/Data/String/Extra.lean

```diff
@@ -33,6 +33,10 @@ opaque fromUTF8Unchecked (a : @& ByteArray) : String
 @[extern "lean_string_to_utf8"]
 opaque toUTF8 (a : @& String) : ByteArray
 
+/-- Accesses a byte in the [UTF-8](https://en.wikipedia.org/wiki/UTF-8) encoding of the `Strig` -/
```

Comment:
```suggestion
/-- Accesses a byte in the [UTF-8](https://en.wikipedia.org/wiki/UTF-8) encoding of the `String`. O(1). -/
```
Not sure we need the UTF-8 reference here :) . It should probably go in the docstring of `String` instead.

Reply:
I only cargo-culted it from `String.toUtf8` (where it is somewhat justifiable). Not sure I
I’d mention it in `String` – it’s an implementation detail, isn’t it?